packages/editor-ui/src/mixins/pushConnection.ts

```diff
@@ -291,6 +291,16 @@ export const pushConnection = defineComponent({
 				}
 			}
 
+			if (receivedData.type === 'workflowActivated') {
+				this.workflowsStore.setWorkflowActive(receivedData.data.workflowId);
+				return true;
+			}
+
+			if (receivedData.type === 'workflowDeactivated') {
+				this.workflowsStore.setWorkflowInactive(receivedData.data.workflowId);
+				return true;
+			}
+
```

Comment:
Might also want to remove this.

Reply:
See above: https://github.com/n8n-io/n8n/pull/7643#discussion_r1393990223